test/scripts/e2e_go_tests.sh

```diff
@@ -7,7 +7,15 @@ set -o pipefail
 
 export GOPATH=$(go env GOPATH)
 export GO111MODULE=on
-GOTESTCOMMAND=${GOTESTCOMMAND:="go test"}
+
+if [ -z "$(which gotestsum)" ]; then
```

Comment:
I think if you swap `test/scripts/run_integration_tests.sh` with `make integration` or `make -o integration` this wouldn't be necessary.

You would just need a way to configure `--junitfile ${TEST_RESULTS}/results.xml` in the makefile

Reply:
will do in a different ticket: https://app.zenhub.com/workspaces/algorand-main-workspace-5ea71fda6d554d53dda36f16/issues/algorand/go-algorand-internal/1426

Reply:
Thanks, in that case just a comment here saying that this is needed because CircleCI isn't using the Makefile